### DISCUSSION TITLE
Make icons across the guide consistent and utilize bitcoinicons.com

### DISCUSSION THREAD

**Discussion by pavlenex** [BitcoinDesign/Guide#295] [Archived]
Posting this on behalf of the user who reviewed the guide.

>  icons from section to section are boxes heading parts but not all the same. some are flat, colored, or have drop shadows. nit picky but would be nice to be uniform.

It would be good to utilize https://bitcoinicons.com through the guide.

![Screenshot 2021-05-07 at 11 33 23](https://user-images.githubusercontent.com/36959754/117430172-25a42880-af28-11eb-988d-7ed57baeae3d.png)
![Screenshot 2021-05-07 at 11 33 30](https://user-images.githubusercontent.com/36959754/117430174-26d55580-af28-11eb-9474-a9fc9c1b03f7.png)
![Screenshot 2021-05-07 at 11 33 38](https://user-images.githubusercontent.com/36959754/117430180-276dec00-af28-11eb-81dc-0f91bd70017b.png)
![Screenshot 2021-05-07 at 11 33 42](https://user-images.githubusercontent.com/36959754/117430182-276dec00-af28-11eb-8248-67964e53026a.png)
![Screenshot 2021-05-07 at 11 34 02](https://user-images.githubusercontent.com/36959754/117430184-28068280-af28-11eb-9a8e-be6ca24cffbe.png)

Opening an issue to see if we consider these an "icons" or graphics, and if we feel that we want them unified in terms of style. Anyhow, may be a good way to add stuff from the bitcoinicons project, thoughts?

**Comment by GBKS** [comment]
I don't think we need to go uniform. Obviously up for discussion, but part of the creative direction of the site was to allow for different types of visual expression, with just a minor set of guidelines. Just like the banner on the home page, I think we should invite different styles, with an eye towards styles that are appropriate for the context.

As for the Bitcoin icons project, I think it's cool to use them if there's overlap. I wouldn't force it though, as the purpose and requirements are quite different. The icon set is highly specific to 20x20 icons on a particular grid to look good at small sizes, while the icons here are more loose (they can be bigger and more detailed and don't need to follow a specific grid). For example, it would be cool to have icons in the style Jessica created for her banner design on some pages, and those wouldn't fit with the icon set at all.

Overall, I think we need more illustrators pushing these graphics and icons to really execute on this concept of many visual expressions. We're not there yet and I personally wouldn't push towards uniformity quite yet.

**Reply by pavlenex** [reply]
Okay, I wasn't aware this was discussed prior. One thing that's not connecting in that statement is that if the Design Guide is there to make stuff consistent and easy, the guide itself should need certain consistency in both visual and textual presentation, though it's a challenge on its own.

In a way, we're making a Bitcoin project/application whose goal is to allow others to build apps easier, so our "Project" should try to dogfood and allow us to follow our own guidelines, principles, etc.

I just like doogfooding, so this was just an idea how we can do it, of course, if the goal is diversity in icons, I'm also fine with that.

**Reply by GBKS** [reply]
I've been repeating my opinion in this, but it would be super good to hear more voices. Don't want to be a dictator and we should do things differently if that's what everyone wants. Consensus and all that...

**Reply by AndyRibetz** [reply]
Correct me if I'm wrong but don't we already have a comprehensive icon set @GBKS and @Bosch-0?

I agree that having different execution of styles, i.e. flat vs dimensional, or color vs black and white may be a little disjointed, but the icon set already made is very aligned and looks as good as any set I've seen in the wild. I think we'd just need to agree to implement them, and/or emulate that style for new icons that may be missing. 

Here, I think I found it... looks great to me! 
![image (5)](https://user-images.githubusercontent.com/83780625/118079145-7b376580-b36c-11eb-85e7-a508e3ad6aa5.png)

My 2 sats.

**Reply by Bosch-0** [reply]
That is the set Pav mentioned in the OP :) We have a site up for the set at https://bitcoinicons.com 

> the guide itself should need certain consistency in both visual and textual presentation, though it's a challenge on its own.

I agree. Visual guidelines / patterns to follow in open-source are very important to have as people are more sporadic with their contributions. If every contributor (some who may submit say only one small graphic) posted custom styled designs things would get very Frankenstein very fast. People who want to jump in and do a small submission also would benefit a lot having some guidelines / direction around what to do visually. In saying that though, there is definitely room for people's personal touch but it's all about finding that balance between not being too rigid whilst still allowing some flexibility.

**Reply by AndyRibetz** [reply]
Holy Schnikeys I'm dum!! Leaving my post unedited as my penance.

**Reply by GBKS** [reply]
Reposting [the video with my thoughts](https://youtu.be/o9apn7hsIEM) on the topic of visual consistency/framework stuff from November.

Also want to point out again that the the set at bitcoinicons.com is meant for use in applications, while the icons in the design guide are illustrative. For example, there's a bandaid icon representing emergency funds in the Personal Finance page, and a icon with two speech bubbles representing user research. Those are not useful for interfaces, and the icon with two speech bubbles has so much detail that it would not fit in the requirements of the Bitcoin icon set (20x20px + grid).

So I'd say that forcing these two different icons sets to be the same does not really work. However, we can still make them look and feel similar. So the Bitcoin icon set does it's thing for UIs and is perfect for that, and then we have the bigger cousin who is clearly part of the same family, but does things just a little different.

And a couple of links that maybe illustrate how I've gone about consistency vs. creative freedom in different aspects of the icons and illustrations:
- [Icons I've created for the guide](https://www.figma.com/file/qzvCvqhSRx3Jq8aywaSjlr/Bitcoin-Design-Guide-Illustrations-CO?node-id=33%3A0)
- [Smaller illustrations with various visual styles](https://www.figma.com/file/qzvCvqhSRx3Jq8aywaSjlr/Bitcoin-Design-Guide-Illustrations-CO?node-id=187%3A236)

Overall though, we don't have the problem of people creating too many different illustrations or going visually crazy. Rather the opposite. I'd just be a bit hesitant restricting things for a problem we may not even have. I'd keep it open and guide things towards consistency in the process of review. It's a nuanced discussion, let's keep digging in.
